test/fixtures_identity_store.js

```diff
@@ -0,0 +1,181 @@
+// Identity Personas
+
+// NURSECONNECT
+    // (+27820001001) - new identity; no subscriptions
+    // (+27820001002) - existing identity with an active MomConnect subscription (no active NurseConnect subscription)
+    // (+27820001003) - existing identity with an active NurseConnect subscription
+
+
+module.exports = function() {
+    return [
+
+        // 0: get identity by msisdn +27820001001
+        {
+            "request": {
+                "url": 'http://is.localhost:8001/api/v1/identities/search/',
+                "method": 'GET',
+                "params": {
+                    "details__addresses__msisdn": '+27820001001'
+                },
+                "headers": {
+                    "key": "get.identity.new.27820001001"
+                }
+            },
+            "response": {
+                "code": 200,
+                "data": {
+                    "count": 0,
+                    "next": null,
+                    "previous": null,
+                    "results": []
+                }
+            }
+        },
+
+        // 1: get identity by msisdn +27820001002
+        {
+            "request": {
+                "url": 'http://is.localhost:8001/api/v1/identities/search/',
+                "method": 'GET',
+                "params": {
+                    "details__addresses__msisdn": '+27820001002'
+                },
+                "headers": {
+                    "key": "get.identity.registered.27820001002"
+                }
+            },
+            "response": {
+                "code": 200,
+                "data": {
+                    "count": 1,
+                    "next": null,
+                    "previous": null,
+                    "results": [{
```

Comment:
I know we've done it like this in the past, but fixture readability is a problem, so rather put the start of the dict on the next line (with an indent).